api-service/src/main/java/com/bitmark/apiservice/middleware/BitmarkApiInterceptor.java

```diff
@@ -31,6 +31,7 @@ public Response intercept(Chain chain) throws IOException {
                 .addHeader("Cache-Control", "no-cache")
                 .addHeader("Cache-Control", "no-store")
                 .addHeader("Accept-Encoding","*")
+                .addHeader("User-Agent", String.format("%s %s %s", "bitmark-sdk-java", System.getProperty("os.name"), System.getProperty("java.version")))
```

Comment:
I am not sure the specification but i think the comma separate each item is better. Ignore if you follow any specification.